docs_user/modules/proc_relocating-one-instance-of-a-monitoring-stack-to-migrate-daemons-to-target-nodes.adoc

```diff
@@ -0,0 +1,179 @@
+[id="relocating-one-instance-of-a-monitoring-stack-to-migrate-daemons-to-target-nodes_{context}"]
+
+= Scenario 2: Relocating one instance of a monitoring stack to migrate daemons to target nodes
```

Comment:
I would remove scenario 2 in the downstream procedure. We should choose a single path, and Scenario 1 fits well this purpose and is easy to follow.
I'm wondering if we can have a conditionally import of this section that describes a different scenario.